src/ir/codegen/gen_equal.go

```diff
@@ -49,7 +49,7 @@ func (g *genEqual) writeCompare(w *bytes.Buffer, pkg *packageData, typ *typeData
 		case "[]ir.Node":
 			fmt.Fprintf(w, "    if !NodeSliceEqual(x.%[1]s, y.%[1]s) { return false }\n", field.Name())
 		case "ir.Doc":
-			fmt.Fprintf(w, "    if x.PhpDocComment != y.PhpDocComment { return false }\n")
+			fmt.Fprintf(w, "    if x.PhpDoc.Raw != y.PhpDoc.Raw { return false }\n")
```

Comment:
(Not related to this PR, but still.)

By the way, it's a little strange that we generate comment-related comparison code for the `equal` method.
Nodes that are identical but have different comments should be equal.
I hope we're not using it when comparing an actual nodes.